http/swagger.yml

```diff
@@ -8938,7 +8938,6 @@ components:
         - shape
         - axes
         - colors
-        - legend
```

Comment:
`staticLegend` doesn't replace it as required?

Reply:
Correct. Static Legend is not required.